app/src/components/datasets/DatasetContentTabs/DatasetContentTabs.tsx

```diff
@@ -15,7 +16,7 @@ const tabs = [
     component: <General />,
   },
   {
-    key: "models",
+    key: DATASET_MODELS_TAB_KEY,
```

Comment:
Since we don't reference this variable anywhere else, I'd actually prefer not defining it. The other tab keys are explicitly defined and exported because we navigate to them from external links.

Reply:
Noted!
Will remove it.